openstack/cinder/templates/vcenter_datacenter_cinder_configmap.yaml

```diff
@@ -55,7 +55,11 @@ template: |
       {{- end }}
       {{ end }}
 
-      [vstorageobject]
+      {{ if .Values.backends.vmware_fcd.enabled | default false }}
+      [vmware_fcd]
       volume_driver = cinder.volume.drivers.vmware.fcd.VMwareVStorageObjectDriver
-      volume_backend_name = vstorageobject
+      volume_backend_name = vmware_fcd
+      vmware_storage_profile = {{ .Values.backends.vmware_fcd.vmware_storage_profile }}
+      extra_capabilities = '{"vcenter-shard": "{= host.split(".")[0] =}"}'
+      {{- end }}
```

Comment:
Why the conditional include and why not do it on `backends.enabled` containing `vmware_fcd`? It doesn't really hurt if there are more config options, I would guess, because only an enabled FCD driver would read them.

Reply:
I presume you mean something like this?

https://github.wdf.sap.corp/cc/secrets/pull/10001

Reply:
yes